packages/eslint-plugin/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@clipboard-health/eslint-plugin",
   "description": "Clipboard's ESLint Plugin",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Devinai as I said, you should not change these. it will be handled by a publish CI jobs.